### PR TITLE
Update new-release.md

### PR DESCRIPTION
**Pull request by marlass**

### CODE REVIEW

**Comment by Zeyber** [comment]
Issues I had when releasing 2.0.0-rc.3:
- Needing to add ANGULAR_CLI_VERSION='9.1' to the config.sh to run `./run.sh install` successfully
- Yarn had a tendency to lock packages (make them read-only) while working on them (I think this was during yarn cache mostly). When this happened, I had to restart my pc in order to have another attempt. This was happening when `./run.sh install` was failing due to the point above.
- pre-release.sh didn't work on windows for me. Instead I needed to run `build:core:lib` and then run `npm run generate:docs` to get the docs. Then I used the ubuntu shell to generate the zip files. Maybe a checklist that breaks these things down in case of script failure?
- I had a lot of issues with packages. Maybe an extra step or at least a note to delete and reinstall node packages in the case of issues.
- Perhaps an extra note to use `dist-tag` under the `Check tags on npm` step in case they are incorrect.

**Comment by marlass** [comment]
> * Needing to add ANGULAR_CLI_VERSION='9.1' to the config.sh to run `./run.sh install` successfully

Let's update the install script to correct version

> * Yarn had a tendency to lock packages (make them read-only) while working on them (I think this was during yarn cache mostly). When this happened, I had to restart my pc in order to have another attempt. This was happening when `./run.sh install` was failing due to the point above.

Making script stable should also take care of this issue

> * pre-release.sh didn't work on windows for me. Instead I needed to run `build:core:lib` and then run `npm run generate:docs` to get the docs. Then I used the ubuntu shell to generate the zip files. Maybe a checklist that breaks these things down in case of script failure?

Polished the Windows steps and provided complete alternative with docs generation and sample data archiving without using the prerelease.sh on Windows

> * I had a lot of issues with packages. Maybe an extra step or at least a note to delete and reinstall node packages in the case of issues.

Same as with yarn issue. Let's make everything work for the first time.

> * Perhaps an extra note to use `dist-tag` under the `Check tags on npm` step in case they are incorrect.

Added that info to the template.

File: .github/ISSUE_TEMPLATE/new-release.md [hunk 1]
```diff
@@ -36,22 +36,22 @@ For Mac:
...
   - [ ] `npm run release:styles:with-changelog`
   - [ ] `npm run release:lib:with-changelog`
   - [ ] `npm run release:core:with-changelog`
   - [ ] `npm run release:cds:with-changelog`
   - [ ] `npm run release:schematics:with-changelog`
 - [ ] Check that release-notes are populated on github (if not, update them)
-- [ ] Check tags on npm
-- [ ] Check libs from a new shell app (install-script) `./run.sh install_npm` (change config's `SPARTACUS_VERSION` to `rc`)
+- [ ] Check tags on npm (`next` tag should always point to the highest version, `latest` for the last stable verion, you can leave `rc` tag until we release stable release). Use `npm dist-tag` for updates.
+- [ ] Check libs from a new shell app (install-script) `./run.sh install_npm` (change config's `SPARTACUS_VERSION` to `next/latest`)
```
Comment (mirkosoft) [inline .github/ISSUE_TEMPLATE/new-release.md]: let's add here a note script has to run within it's root: `cd scripts/install`

File: .github/ISSUE_TEMPLATE/new-release.md [hunk 2]
```diff
@@ -36,22 +36,22 @@ For Mac:
...
-- [ ] Download and rename in root directory `https://github.tools.sap/cx-commerce/spartacussampledataaddon/archive/develop.tar.gz` -> `spartacussampleaddon.tar.gz`
+- [ ] Cleanup repo, build and generate compodocs and publish on github pages (`yarn generate:docs` and `yarn publish:docs` for patch stable/releases)
+- [ ] Download and rename in root directory `https://github.tools.sap/cx-commerce/spartacussampledataaddon/archive/develop.zip` -> `spartacussampledataaddon.zip`
+- [ ] Download and rename in root directory `https://github.tools.sap/cx-commerce/spartacussampledataaddon/archive/develop.tar.gz` -> `spartacussampledataaddon.tar.gz`
 
 ---
 
-- [ ] Release libraries with release-it scripts (set GITHUB_TOKEN env variable, do `npm login`; for each package select rc1 version when prompted - when prompted):
+- [ ] Release libraries with release-it scripts (set GITHUB_TOKEN env variable, check if you are loggin in with `npm whoami` (if not logged, then login with `npm login`); for each package select version when prompted):
```
Comment (mirkosoft) [inline .github/ISSUE_TEMPLATE/new-release.md]: if version is not suggested it's needed to type it; so let's change the sentence to: "for each package select/type version when prompted"

**Review by mirkosoft — CHANGES_REQUESTED** [review]
It's great, just minor changes proposed.

**Review by mirkosoft — APPROVED** [review]
OK!

**Pull request merged**